.github/workflows/main.yml

```diff
@@ -98,6 +98,7 @@ jobs:
           - "uzh"
           - "vai"
           - "vsc_ugent"
+          - "crg"
```

Comment:
If I am not wrong this list is alphabetically ordered, take a look

Reply:
Yes, you are right